### DISCUSSION TITLE
How to generate test run link in automated way

### DISCUSSION THREAD

**Discussion by abhayshankar** [ludeknovy/jtl-reporter#133] [Q&A]
Is there any way to generate the sharable test run link in an automated way ?
I want to generate and share the test run link via email once the load test completes and jtl file is uploaded in jtlReporter.

**Comment by ludeknovy** [comment]
Hi @abhayshankar! There is an endpoint for generating the token, but I was thinking about it lately and maybe the token could be generated on uploading new test runs - there could be setting a flag for it with on/off as possible values.

But you will need to create the url on your own, but that's very easy.

What do you think?

**Reply by abhayshankar** [reply]
Sounds good to me. Let me try the existing endpoint.

**Reply by ludeknovy** [reply]
See: https://github.com/ludeknovy/jtl-reporter-be/blob/master/src/server/routes/item.ts#L86

Do not forget to add `x-access-token` header with valid api key.

**Reply by ludeknovy** [reply]
https://github.com/ludeknovy/jtl-reporter/issues/136
